lib/engine/game/g_1860.rb

```diff
@@ -223,6 +255,125 @@ def close_other_companies!(company)
         @companies.reject { |c| c == company }.each(&:close!)
         @log << '-- Event: starting private companies close --'
       end
+
+      def biggest_train(corporation)
+        corporation.trains.map(&:distance).max || 0
```

Comment:
trains.max(&:distance)

Reply:
Ok

Reply:
biggest_train needs to return an integer, not a train.

Reply:
this can go in the step, it's not used anywhere else it looks like